CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
```

Comment:
@stephenmcm, is there anything that needs adding here for someone that's bumping SS3 to SS4?